client/blocks/plugins-scheduled-updates-multisite/schedule-create.tsx

```diff
@@ -1,14 +1,21 @@
 import { Button } from '@wordpress/components';
 import { close, Icon } from '@wordpress/icons';
 import { useTranslate } from 'i18n-calypso';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
 import { ScheduleForm } from './schedule-form';
+import type { MultiSiteSuccessParams } from 'calypso/blocks/plugins-scheduled-updates-multisite/types';
 
 type Props = {
 	onNavBack?: () => void;
 };
 
 export const ScheduleCreate = ( { onNavBack }: Props ) => {
 	const translate = useTranslate();
+
+	const onRecordSuccessEvent = ( params: MultiSiteSuccessParams ) => {
+		recordTracksEvent( 'calypso_scheduled_updates_multisite_create_schedule', params );
```

Comment:
Should we also track create/edit/delete for the individual sites? Or no?

Reply:
> Should we also track create/edit/delete for the individual sites? Or no?

Hmm, good question. If we're using frontend events as the single source of truth, I'll say we need to make additional tracks for individual sites too. Otherwise, we'll need to add tracks in our Jetpack repo. Since we already have the frontend events from the very beginning, I'll handle it in the frontend and record those into individual sites too, but please let me know if you prefer to record it in Jetpack repo.

Reply:
Update: it now fires a multisite level track event, followed by individual site events based on the action (create/edit/delete).